hudi-spark-datasource/hudi-spark/src/main/scala/org/apache/hudi/HoodieSparkSqlWriter.scala

```diff
@@ -135,6 +135,7 @@ private[hudi] object HoodieSparkSqlWriter {
       // scalastyle:off
       if (parameters(ENABLE_ROW_WRITER_OPT_KEY).toBoolean &&
         operation == WriteOperationType.BULK_INSERT) {
+        log.info("Using BULK_INSERT")
```

Comment:
is this really required?

Reply:
Yes as of now there is no easy way to tell that bulk insert is being used unless you look at the query plan and then reference the codebase.